eng/common/scripts/Detect-Api-Changes.ps1

```diff
@@ -22,8 +22,13 @@ Param (
 $configFileDir = Join-Path -Path $ArtifactPath "PackageInfo"
 
 # Submit API review request and return status whether current revision is approved or pending or failed to create review
-function Submit-Request($filePath, $packageName, $packageType)
+function Submit-Request($filePath, $packageName, $packageType, $group)
 {
+    # Construct full package name with groupId if available
+    $fullPackageName = $packageName
+    if ($group) {
+        $fullPackageName = "${group}:$packageName"
```

Comment:
Would it be better to do this on the java side before calling this method instead of adding the group parameter here?